cluster/config-defaults.yaml

```diff
@@ -1194,4 +1194,8 @@ teapot_admission_controller_scheduling_controls_enabled: "false"
 teapot_admission_controller_scheduling_controls_default_architecture: "amd64"
 
 # role-sync-controller configs
+{{ if eq .Cluster.Environment "e2e" }}
+role_sync_controller_enabled: "true"
```

Comment:
This would enable the sync controller for non-EKS clusters. Do we want that?

Reply:
Hmm I could change it to EKS only?

My plan is to ultimately keep the `role-sync-controller` enabled and disable the authz webhook at some point, for all clusters. This allows us:
1. single authz setup with no parity between eks/non-eks
2. test the setup by rolling it out on channels and fix issues if we find any

We can hold off on that while we still have some work pending on the admission-controller, I guess? What do you suggest?

Reply:
I'm fine with that plan.

But with this, it would create additional bindings in the namespaces that we don't need because auth-webhook is still running. I'm just concerned about confusion (it should be fine from a permissions perspective).